nipype/pipeline/engine/tests/test_utils.py

```diff
@@ -157,70 +157,3 @@ def test_provenance(tmpdir):
     psg = write_workflow_prov(eg, prov_base, format='all')
     assert len(psg.bundles) == 2
     assert len(psg.get_records()) == 7
-
-
-def dummy_func(value):
-    return value + 1
-
-
-@pytest.mark.skipif(
-    sys.version_info < (3, 0), reason="the famous segfault #1788")
-def test_mapnode_crash(tmpdir):
-    """Test mapnode crash when stop_on_first_crash is True"""
-    cwd = os.getcwd()
-    node = pe.MapNode(
-        niu.Function(
-            input_names=['WRONG'],
```

Comment:
@satra These tests depend on bad input names. I don't know if they can be rewritten to test the segfaulting code path, or disallowing `input_name` mismatches prevents it entirely.